package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "enclave",
-  "version": "0.10.1",
+  "version": "0.10.2",
```

Comment:
Don't worry about doing this, there's a release script we run off master which does this programmatically